docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#stub-response-headers' %}
+
+### Stub Response Headers
+
+Before it passes an incoming response to the application, `RequestHook` can change the response headers.
```

Comment:
`Before it passes an incoming response to the application` - what  is it?

> effectively mocking part of the response.
You don't mock part of the response - you change the existing response.

I propose to write the main idea at once:
```js
You can use the `RequestHook._onConfigureResponse()` method to set the header value or remove existing header.
```

Reply:
See [comment](https://github.com/DevExpress/testcafe/pull/6089#discussion_r603547525)